charts/oes/values.yaml

```diff
@@ -560,7 +560,7 @@ visibility:
   image:
     registry: quay.io/opsmxpublic
     repository: ubi8-oes-visibility
-    tag: v3.8.0
+    tag: v3.8
```

Comment:
why did we change this? it should be v3.8.x

Reply:
fixed it